frontend/src/App.tsx

```diff
@@ -8,6 +8,9 @@ import UnAuthorizedPage from "./pages/UnAuthorizedPage/UnAuthorizedPage";
 import OAuth2RedirectHandler from "./pages/OAuth2RedirectHandler/OAuth2RedirectHandler";
 import Profile from "./pages/Profile/Profile";
 import Login from "./pages/Login/Login";
+import Cart from "./pages/Cart/Cart";
+import OrderHistory from "./components/OrderHistory/OrderHistory";
+import SimulatedPayment from "./components/SimulatedPayment/SimulatedPayment";
```

Comment:
please use absolute `src/` to maintain with other files

Reply:
Updated imports to be absolute